app/models/katello/concerns/content_facet_host_extensions.rb

```diff
@@ -53,6 +53,13 @@ def find_by_installable_errata(_key, operator, value)
             { :conditions => "#{::Host::Managed.table_name}.id IN (#{facets.pluck(:host_id).join(',')})" }
           end
         end
+
+        def in_content_view_environment(content_view = nil, lifecycle_environment = nil)
```

Comment:
If both are nil, the joins() actually restricts to any hosts that have a content_view_environment.  This may be useful, but a user of the function may not know what to expect with that input.  I'd probably rather error if both are nil.  Thoughts?